boards/others/esp32c3_supermini/doc/index.rst

```diff
@@ -0,0 +1,287 @@
+.. _esp32c3_supermini:
+
+ESP32-C3-SUPERMINI
+##################
+
+Overview
+********
+
+ESP32-C3-SUPERMINI is based on the ESP32-C3, a single-core Wi-Fi and Bluetooth 5 (LE) microcontroller SoC,
+based on the open-source RISC-V architecture. This board also includes a Type-C USB Serial/JTAG port.
+For more information, check `ESP32-C3-SUPERMINI`_
+
+.. figure:: img/esp32c3_supermini.webp
+     :align: center
+     :alt: An image of the a typical ESP32-C3 SuperMini development board
+      ESP32-C3 SuperMini board (Credit: https://blakadder.com)
```

Comment:
please use `zephyr:board` directive (see in other ESP32 boards), which means you also drop the figure as it will automagically be pulled in by the directive.

Reply:
"automagically", 1st time reading this, haha, great!